Bugsnag/Client/BugsnagClient+Private.h

```diff
@@ -138,6 +138,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)start;
 
+/// Alters whether error detection should be enabled or not after Bugsnag has been initialized.
+/// Intended for internal use only by React Native/Unity.
+@property (nonatomic) BOOL autoNotify;
+
```

Comment:
The properties are defined before the methods in this header, and alphabetically - please move this up to line 38

Reply:
Moved property into alphabetical order.